llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -767,16 +767,11 @@ void InterBlockScheduling::emitBundles(
       TII->insertNoop(*BB, Before);
       continue;
     }
-    bool InBundle = false;
     for (auto *MI : Bundle.getInstrs()) {
       if (Move) {
         BB->remove_instr(MI);
       }
       BB->insert(Before, MI);
-      if (InBundle) {
-        MI->bundleWithPred();
-      }
-      InBundle = true;
```

Comment:
CHECK: bundling will now be done when processing the inserted region in its parent's block

Reply:
This was just redundant. the `AIEHazardRecognizer::applyBundles()` call below does the bundling as well. (And finalizes them)